### DISCUSSION TITLE
english document require

### DISCUSSION THREAD

**Discussion by yupix** [yupix/Mi.py#64] [Show and tell]
I'm Japanese and I'm not good at English, so I rely on translation, so the maintenance of the document is inevitably delayed. If anyone can help me, I would appreciate it if you could send me a pull request etc.

**Comment by fotoente** [comment]
Unfortunatley i can't speak japanese, but with deepl  mabe i can get a starting point, someone with better language skills can provide a better solution... Just gimme a bit of time

**Reply by yupix** [reply]
I changed it so that I can translate the document into multiple languages. 
https://yupix.github.io/Mi.py/en/mi.html
